core/jobs/batch_jobs/suggestion_stats_computation_jobs.py

```diff
@@ -47,14 +50,14 @@
     from mypy_imports import opportunity_models
     from mypy_imports import suggestion_models
 
-(opportunity_models, suggestion_models) = models.Registry.import_models([
-    models.Names.OPPORTUNITY, models.Names.SUGGESTION
+(opportunity_models, suggestion_models, topic_models) = models.Registry.import_models([
+    models.Names.OPPORTUNITY, models.Names.SUGGESTION, models.Names.TOPIC
 ])
 
 datastore_services = models.Registry.import_datastore_services()
 
 
-class TranslationContributionsStatsDict(TypedDict):
+class TranslationStatsDict(TypedDict):
```

Comment:
```suggestion
class ContributionStatsDict(TypedDict):
```

Since this also includes question stats.

Reply:
Done